src/topology/connected.lean

```diff
@@ -132,6 +133,150 @@ begin
     Hs.is_preconnected Ht.is_preconnected
 end
 
+/--
+The union Union s of an increasing mapping s of preconnected sets
```

Comment:
Is there a typo here? If not, how should I interpret "The union Union s"?

Reply:
You might also want to format using ``:
```suggestion
The union `Union s` of an increasing mapping `s` of preconnected sets
```
but `Union s` doesn't make sense regardless.

Reply:
What do you mean by using the ``⋃``notation?